lily4178993/maths-magicians
# Implement Calculator UI

# Implement Calculator UI

## Description
This pull request includes the implementation of the Calculator UI as per the project requirements. The following tasks have been completed:

- Deleted all the boilerplate from Create React App.
- Created a new directory called `components`.
- Inside `components`, created a new `Calculator.js` file.
- Implemented the design of the Calculator UI (JSX and CSS).
- Utilized functional components and created at least 1 child component.
- Integrated the `Calculator` component into the top-level component `App.js`.

## Screenshots

<img width="256" alt="calculator" src="https://github.com/lily4178993/maths-magicians/assets/101261047/4bb6e5f4-c3b8-4c70-bd53-2c1464ba781e">

## Checklist
- [x] Deleted Create React App boilerplate.
- [x] Created `components` directory.
- [x] Implemented Calculator UI.
- [x] Used functional components.
- [x] Created at least 1 child component.
- [x] Integrated `Calculator` into `App.js`.

## Reviewer Notes
Please review the implementation of the Calculator UI and ensure it matches the provided design screenshot. The math logic for the calculator will be added in the next project phase.

## How to Test
To test the Calculator UI, follow these steps:

1. Ensure you have cloned this branch and installed dependencies.
2. Run the development server:
   ```bash
   npm start
   ```
3. Open your web browser and access the application to interact with the Calculator UI.
